lib/index.js

```diff
@@ -17,7 +17,7 @@ const optionsSchema = Joi.object({
     validatorUrl: Joi.string().allow(null, false).optional(),
     docExpansion: Joi.string().allow(null).valid(['none', 'list', 'full']).optional(),
     defaultModelRendering: Joi.string().allow(null).valid(['model', 'schema']).optional(),
-    apisSorter: Joi.string().allow(null).valid(['none', 'alpha']).optional(),
+    apisSorter: Joi.alternatives().try(Joi.array().items(Joi.string()), Joi.string().allow(null).valid(['none', 'alpha'])).optional(),
```

Comment:
Ah cool - thanks for pointing that out, good to know.

For this use case, the string validation is different depending on whether or not it's in the array.  If not an array it is an enum, however if an array it's an array of tags which could be any non-empty string